build-scripts/puddle-conf/errata-puddle-4.1-signed.el7.conf

```diff
@@ -0,0 +1,18 @@
+[puddle]
```

Comment:
Are puddle config files like `errata-puddle-x.y.conf` `errata-puddle-x.y-signed.conf` still needed?
They don't seem to be used in the signed-compose job.

Reply:
They're cruft. I thought I might keep them around in case we need to build a compose the old way, but... more likely just to confuse things.